pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -779,13 +779,17 @@ cdef class _Timedelta(timedelta):
                     return PyObject_RichCompare(np.array([self]), other, op)
                 return PyObject_RichCompare(other, self, reverse_ops[op])
             else:
-                if op == Py_EQ:
-                    return False
-                elif op == Py_NE:
-                    return True
-                raise TypeError('Cannot compare type {cls} with type {other}'
-                                .format(cls=type(self).__name__,
-                                        other=type(other).__name__))
+                if other is NaT:
+                    return PyObject_RichCompare(other, self, reverse_ops[op])
+                else:
```

Comment:
you don't need the else here

Reply:
or can do an elif

Reply:
done, if you want me to put it at the end of the `if`s for the minor efficiency gain I can do that as well.

Reply:
it’s all about readability ; efficiency is way down the list